packages/frontend/src/state/connector/index.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import React, { createContext, ReactNode } from "react";
+import {
+    createDispatchHook,
+    createSelectorHook,
+    createStoreHook,
+    Provider,
+    ReactReduxContextValue,
+} from "react-redux";
+import { store } from "../";
+
+export const HostStateContext = createContext<ReactReduxContextValue>(
+    // I know... super ugly... but: https://redux-toolkit.js.org/rtk-query/usage/customizing-create-api#customizing-the-react-redux-hooks
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    null as any
+);
+
+export const useStore = createStoreHook(HostStateContext);
+export const useDispatch = createDispatchHook(HostStateContext);
+export const useSelector = createSelectorHook(HostStateContext);
+
+interface HostStateProviderProps {
+    children: ReactNode;
+}
+
+export const HostStateProvider = ({ children }: HostStateProviderProps) => {
```

Comment:
why the name `HostStateProvider`?

Could find out it was a `redux` provider.

Reply:
If the provider is from the current repo, I think I would like more something like `CurrentStateProvider`.

But I'm okay with Host as well.